cmd/utils_test.go

```diff
@@ -354,3 +354,37 @@ func Test_sliceContains_Fail(t *testing.T) {
 		t.Errorf("Slice a should not have b")
 	}
 }
+
+func Test_generateGraph(t *testing.T) {
+
+	/*
+
+	         A
+	       / | \
+	      /  |  \
+	     G   B   D
+	     |\  |   /\
+	     | \ |  /  \
+	     F   C      E
+
+	 A is the main Module here
+	 */
+	//x@version format signifies x-->module name,version--> version of module
+	goModGraphOutputString := "A@1.1 G@1.2\nA@1.1 B@1.3\nA@1.1 D@1.2\nG@1.5 F@1.3\nG@1.5 C@1.1\nB@1.3 C@1.1\nD@1.2 C@1.1\nD@1.2 E@1.8"
```

Comment:
Could you please replace the double quotes with [string literals](https://golang.org/ref/spec#String_literals)? You won't need the `\n` then and it would look cleaner to read.